tests/Functional/Types/DateTimeTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Tests\Functional\Types;
+
+use Doctrine\DBAL\Schema\Table;
+use Doctrine\DBAL\Tests\FunctionalTestCase;
+use Doctrine\DBAL\Types\Type;
+use Doctrine\DBAL\Types\Types;
+
+use function abs;
+use function get_class;
+use function sprintf;
+
+final class DateTimeTest extends FunctionalTestCase
+{
+    /** @return string[][] */
+    public function dataValuesProvider(): array
```

Comment:
Please make sure to declare all data providers `static`.

Reply:
Done in a new patch.